libmscore/chordrest.cpp

```diff
@@ -843,7 +843,10 @@ QVariant ChordRest::propertyDefault(Pid propertyId) const
             default:
                   return DurationElement::propertyDefault(propertyId);
             }
+#if (!defined (_MSCVER) && !defined (_MSC_VER))
+      // Prevent unreachable code warning in MSVC
```

Comment:
If that code is unreachable irrespectively of compiler used, remove it altogether